server/public/styles.css

```diff
@@ -0,0 +1,80 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: auto 0;
+}
+
+i {
+  font-size: 1.25rem;
+  filter: drop-shadow(3px 3px 2px grey);
+}
+
+img {
+  object-fit: contain;
+  max-height: 10rem;
+}
+
+.title {
+  text-align: center;
+}
+
+.title > h1 {
+  background: radial-gradient(50% 50% at 50% 50%, #c5f9fa 25%, rgba(255, 255, 255, 0) 100%), #66acad;
+  font-family: 'Almendra SC', serif;
+  font-size: 2rem;
+  padding: 1rem;
+  margin: auto;
+}
+
+i:hover,
+h3:hover,
+h4:hover {
+  cursor: pointer;
+}
+
+.container.nav {
+  display: flex;
+  justify-content: space-around;
+  padding: 1rem;
+  background-color: #c5f9fa;
+}
+
+.search {
+  margin-right: 1rem;
+  padding-left: 0.25rem;
+  border-radius: 50px;
+  border: 0 white solid;
+}
+
+.menu {
+  background-color: white;
+  padding: 1rem;
+}
+
+.theShade {
+  cursor: pointer;
+  width: 100vw;
+  height: 100vh;
+  background-color: rgba(169, 169, 169, 0.5);
+}
+
+.hidden {
+  display: none;
+}
+
+.row {
+  margin: 1rem;
+}
+
+.card {
+  filter: drop-shadow(3px 3px 2px grey);
+  background-color: rgb(245, 245, 245);
+  text-align: center;
+  margin-bottom: 1rem;
+}
+
+.card-body {
+  padding: 0.25rem 0;
+}
```

Comment:
Bootstrap has text alignment and spacing utilities you should leverage. Also, avoid adding styles to library classes.

https://getbootstrap.com/docs/4.5/utilities/spacing/
https://getbootstrap.com/docs/4.5/utilities/text/